spec.bs

```diff
@@ -632,7 +642,7 @@ The [=steps to respond to a payment request=] for this payment method, for a giv
     : {{AuthenticationExtensionsPaymentInputs/isPayment}}
     :: The boolean value `true`.
     : {{AuthenticationExtensionsPaymentInputs/rp}}
-    :: **TODO**
+    :: |data|["{{SecurePaymentConfirmationRequest/rpId}}"]
```

Comment:
Please remove the TODO on line 655 below.

Reply:
Done